Node.gitignore

```diff
@@ -16,6 +16,7 @@ lib-cov
 
 # Coverage directory used by tools like istanbul
 coverage
+coverage.lcov
```

Comment:
Could this be generalized to `*.lcov` if it's a file rather than a folder?

Reply:
Okey, I'll do that :)